composite/proof-of-reserves/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "dist",
+    "rootDir": "src",
+    "typeRoots": ["../../node_modules/@types", "../../typings", "./typings"]
+  },
+  "include": ["src/**/*"],
```

Comment:
```suggestion
  "include": ["src"],
```

Reply:
Took this from [here](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html), we may not need it for this flat structure but some refactor in the future could introduce subfolders and break things, so I would just leave it as is.

Reply:
it actually works recursively even without the glob

Reply:
This is fine though, we can address it later